apps/desktop-wallet/src/api/queries/addressQueries.ts

```diff
@@ -94,7 +97,8 @@ export type AddressTokensSearchStringQueryFnData = {
 // Generates a string that includes the names and symbols of all tokens in the address, useful for filtering addresses
 export const addressTokensSearchStringQuery = ({ addressHash, networkId }: AddressLatestTransactionQueryProps) =>
   queryOptions({
-    queryKey: ['address', addressHash, 'tokensSearchString', { networkId }],
+    queryKey: ['address', addressHash, 'computedData', 'tokensSearchString', { networkId }],
+    ...getQueryConfig({ staleTime: Infinity, gcTime: Infinity, networkId }),
```

Comment:
Calculating the search string is an expensive operation (consider that the search string of an address is a string that includes token names, symbols, nft names, etc). We don't want to flush this cache until it's necessary (which is when the balances of the address change)